### PR TITLE
Add in UV topology display option

### PR DESCRIPTION
**Pull request by bernardkwok**
Update for  #426.
Allow for uv topology to be seen to avoid having to go back to a DCC to view it.

### CODE REVIEW

File: source/MaterialXView/Viewer.cpp [hunk 1]
```diff
@@ -1088,11 +1094,145 @@ void Viewer::drawContents()
...
+                                                 unsigned int index,
+                                                 const std::string& positionStreamName)
+{
+    mx::MeshStreamPtr uvStream3D = mesh->getStream(positionStreamName);
+    if (!uvStream3D)
+    {
+        uvStream3D = mx::MeshStream::create(positionStreamName, mx::MeshStream::POSITION_ATTRIBUTE, 0);
+        mesh->addStream(uvStream3D);
+
+        mx::MeshStreamPtr uvStream2D = mesh->getStream(uvStreamName, index);
```
Comment (niklasharrysson) [inline source/MaterialXView/Viewer.cpp]: Can we assume this stream is always available? Or do we need a nullptr check on uvStream2D?

File: source/MaterialXView/Viewer.cpp [hunk 2]
```diff
@@ -1088,11 +1094,145 @@ void Viewer::drawContents()
...
+        }
+        catch (std::exception& e)
+        {
+            _wireMaterialUV = nullptr;
+            std::cerr << "Failed to generate uv wire shader: " << e.what();
+            return;
+        }
+    }
+
+    GLShaderPtr shader = _wireMaterialUV ? _wireMaterialUV->getShader() : nullptr;
```
Comment (niklasharrysson) [inline source/MaterialXView/Viewer.cpp]: Null check on _wireMaterialUV is already done above.

File: source/MaterialXView/Viewer.cpp (hunk 1 above)
Comment (bernardkwok) [reply source/MaterialXView/Viewer.cpp]: We should probably check at the top as it's useless to display when there are no uvs. WIll fix.

File: source/MaterialXView/Viewer.cpp (hunk 2 above)
Comment (bernardkwok) [reply source/MaterialXView/Viewer.cpp]: Will fix.

**Pull request merged**